sentry_sdk/integrations/django/__init__.py

```diff
@@ -357,6 +359,38 @@ def sentry_patched_get_response(self, request):
         patch_get_response_async(BaseHandler, _before_get_response)
 
 
+def patch_templates():
+    # type: () -> None
+    import django.shortcuts
+    from django.template.response import SimpleTemplateResponse
+
+    real_render = django.shortcuts.render
+    real_rendered_content = SimpleTemplateResponse.rendered_content
+
+    @_functools.wraps(real_render)
+    def render(request, template_name, context=None, *args, **kwargs):
+        hub = Hub.current
+        if hub.get_integration(DjangoIntegration) is None:
+            return real_render(request, template_name, context, *args, **kwargs)
+
+        with hub.start_span(op="django.render", description=template_name) as span:
```

Comment:
I think the span name is too nondescriptive, would go with django.template.render

Reply:
The mapping is the rendering of an HTTP response specifically, not rendering in generic context so it works with any template backend, and avoid flooding as django do recursive rendering.

I'm updating according to your suggestion.